### DISCUSSION TITLE
Move Spectrogram under admin tools

### DISCUSSION THREAD

**Discussion by MABeatty1978** [mcguirepr89/BirdNET-Pi#1005] [Ideas]
I just went through all the steps to make my install public and am publicly sharing it.  Got it all going and all is well....until..... I realized that the spectrograph plays a live recording and I have the microphone for this on my back yard patio.  This enables anyone in the world with an internet connection to hear every word I say in my back yard.

I think it would be good to move this to a  password protected area of BirdNet.

**Comment by srd424** [comment]
I've just managed to reproduce this here, but that looks to be a bug in my docker images, which I'm guessing you're not using?

The 'raw' live stream URL (`http://<your-host>/stream`) should be protected by Caddy, and the caddy config file gets rewritten when you set a password. Have you checked from incognito mode or another device to make sure you're not sending a previously saved password?

**Reply by srd424** [reply]
(I've pushed a fix for the bug in my docker images, incidentally!)

**Reply by MABeatty1978** [reply]
My apologies.  I just had the login cached so it was not prompting.  Switching to incognito confirmed the authentication is working.  Thank you for the reply.
